packages/component-meta/scripts/generateComponentMeta.ts

```diff
@@ -53,8 +53,9 @@ const generateComponentMeta = (): void => {
   requiredProps?: string[]; // array of props that are mandatory
   deprecatedProps?: string[]; // array of props that are deprecated
   breakpointCustomizableProps?: string[]; // array of props that are breakpointCustomizable
+  arrayProps?: string[]; // array of props that are of type array
   allowedPropValues?: {
-    [propName: string]: 'boolean' | 'number' | 'string' | object | string[];
+    [propName: string]: 'boolean' | 'number' | 'string' | object | string[] | number[] | boolean[];
```

Comment:
I think this isn't relevant anymore since `['string']` seems to be the final result which is covered by `string[]` type.
I guess that format is okay, perhaps specifying it as `'string[]'` here would have made things easier to use and understand but not sure.